packages/button/src/index.ts

```diff
@@ -1,9 +1,10 @@
-import { FrameworkConfiguration, PLATFORM } from 'aurelia-framework';
+import { FrameworkConfiguration, DOM } from 'aurelia-framework';
+import { UxButton } from './ux-button';
+import css from './ux-button.css';
 
 export { UxButtonTheme } from './ux-button-theme';
 
 export function configure(config: FrameworkConfiguration) {
-  config.globalResources([
-    PLATFORM.moduleName('@aurelia-ux/button/ux-button')
-  ]);
+  DOM.injectStyles(css, undefined, undefined, 'ux-button-css');
```

Comment:
So I am not sure how I feel about the style injection happening in the index page. This feels like it would be a better fit in the components view-model, either in a custom decorator or just in the constructor.

The ideal scenario would have been figuring out why the `require` tag is not working in Webpack when using rollup to move everything into a single file.

Reply:
We can copy the `.css` files over to dist to have it dynamically bundled. It will work fine, we just need to adjust the test to have those `css` file properly mapped